proto/qmstr/service/reportservice.proto

```diff
@@ -9,9 +9,8 @@ message ReporterConfigRequest {
 }
 
 message ReporterConfigResponse {
-  string session = 1;
-  map<string, string> configMap = 2;
-  string name = 4;
+  map<string, string> configMap = 1;
```

Comment:
Field numbers should not be changed for backward compatibility.

Reply:
Since we are still in an early stage and there are not many dependencies on the proto files we can still change the field numbers.